tests/TestCase/IntegrationTest.php

```diff
@@ -55,6 +55,6 @@ public function testLocalizedProviderIntegration()
             'phoneField' => ['myCustomRuleNameForPhone' => 'Numéro invalide'],
             'postalField' => ['myCustomRuleNameForPostal' => 'The provided value is invalid'],
         ];
-        $this->assertEquals($expected, $errors);
+        $this->assertSame($expected, $errors);
```

Comment:
Careful with arrays
Here then order also matters which is sometimes not what we want to assert on in tests.

Reply:
Yes. You're right. Maybe we can split `assertSame` to many array assertions?

For example, we can consider using the `assertArrayHasKey` and `assertSame` to assert the value with `$errors['phoneField']`.

What do you think about this?

Reply:
Probably not worth it here, as the result is an array of nonempty strings. So not much that can go wrong usually.

Reply:
Ok. Revert to `assertEquals`.